src/azure-cli-core/tests/test_rp_auto_reg.py

```diff
@@ -12,23 +12,23 @@
 from azure.cli.core.commands import _check_rp_not_registered_err
 
 
-@record_only()
-class TestRPAutoRegister(ScenarioTest):
-    def setUp(self):
-        if self.in_recording:
-            # note, unregistering a registered provider might take a long while(10+ mins)
-            result = self.cmd('provider show -n Microsoft.Sql --query "registrationState" -o tsv').output
-            if result.lower().strip() != 'unregistered':
-                self.cmd('provider unregister -n Microsoft.Sql --wait')
-                time.sleep(30)  # a bit random but like to ensure unregistering went through
-
-        return super(TestRPAutoRegister, self).setUp()
-
-    @ResourceGroupPreparer()
-    def test_rp_auto_register(self, resource_group, resource_group_location):
-        cmd = ('sql server create -g {} -n ygserver123 -l {} --admin-user sa123 --admin-password verySecret12345')
-        self.cmd(cmd.format(resource_group, resource_group_location), checks=JMESPathCheck('name', 'ygserver123'))
-        self.cmd('provider show -n Microsoft.Sql', checks=JMESPathCheck('registrationState', 'Registered'))
+# @record_only()
+# class TestRPAutoRegister(ScenarioTest):
+#     def setUp(self):
+#         if self.in_recording:
+#             # note, unregistering a registered provider might take a long while(10+ mins)
+#             result = self.cmd('provider show -n Microsoft.Sql --query "registrationState" -o tsv').output
+#             if result.lower().strip() != 'unregistered':
+#                 self.cmd('provider unregister -n Microsoft.Sql --wait')
+#                 time.sleep(30)  # a bit random but like to ensure unregistering went through
+
+#         return super(TestRPAutoRegister, self).setUp()
+
+#     @ResourceGroupPreparer()
+#     def test_rp_auto_register(self, resource_group, resource_group_location):
+#         cmd = ('sql server create -g {} -n ygserver123 -l {} --admin-user sa123 --admin-password verySecret12345')
+#         self.cmd(cmd.format(resource_group, resource_group_location), checks=JMESPathCheck('name', 'ygserver123'))
+#         self.cmd('provider show -n Microsoft.Sql', checks=JMESPathCheck('registrationState', 'Registered'))
```

Comment:
Thanks for the change @jaredmoo Actually you can just delete these code. I just confirmed with @yugangw-msft , it's good to go.